doc/api/buffer.md

```diff
@@ -3589,3 +3595,4 @@ introducing security vulnerabilities into an application.
 [binary strings]: https://developer.mozilla.org/en-US/docs/Web/API/DOMString/Binary
 [endianness]: https://en.wikipedia.org/wiki/Endianness
 [iterator]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
+[`v8::TypedArray::kMaxLength`]: https://v8.github.io/api/head/classv8_1_1TypedArray.html#a54a48f4373da0850663c4393d843b9b0
```

Comment:
Shouldn't this be placed right after the `util.inspect()` link?

Reply:
Thanks, it seems the linter in my local environment did not work😂